core/src/main/java/com/flipkart/varadhi/core/cluster/entities/ShardDlqMessageRequest.java

```diff
@@ -0,0 +1,30 @@
+package com.flipkart.varadhi.core.cluster.entities;
+
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.flipkart.varadhi.entities.Offset;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@AllArgsConstructor(onConstructor = @__(@JsonCreator))
+public class ShardDlqMessageRequest {
+    private long earliestFailedAt = 0L;
+    private final String pageMarker;
+    private int limit = -1;
```

Comment:
what would limit = -1 mean?

Reply:
that was to indicate default value, but not a valid scenraio.
removed default initialization.